patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -5827,6 +5928,31 @@
  			try {
  				RijndaelManaged rijndaelManaged = new RijndaelManaged();
  				rijndaelManaged.Padding = PaddingMode.None;
+@@ -38410,16 +_,24 @@
+ 
+ 							player.statLife = binaryReader.ReadInt32();
+ 							player.statLifeMax = binaryReader.ReadInt32();
++							// Allow health stat to exceed vanilla bounds (#HealthManaAPI)
++							/*
+ 							if (player.statLifeMax > 500)
+ 								player.statLifeMax = 500;
++							*/
+ 
+ 							player.statMana = binaryReader.ReadInt32();
+ 							player.statManaMax = binaryReader.ReadInt32();
++							// Allow mana stat to exceed vanilla bounds (#HealthManaAPI)
++							/*
+ 							if (player.statManaMax > 200)
+ 								player.statManaMax = 200;
+ 
+ 							if (player.statMana > 400)
+ 								player.statMana = 400;
++							*/
```

Comment:
Only one of these you need to comment out is the `statMana`, as the saved values here are vanilla max values anyway, so no problem with capping them to vanilla